tests/unit/test_charm.py

```diff
@@ -322,20 +403,20 @@ class TestDelayedPebbleReady(unittest.TestCase):
     def setUp(self):
         # Patch _check_alert_rules, which attempts to talk to a loki server endpoint
         self.check_alert_rules_patcher = patch(
-            "charms.loki_k8s.v0.loki_push_api.LokiPushApiProvider._check_alert_rules",
-            new=tautology,
+            "charm.LokiOperatorCharm._check_alert_rules",
+            new=lambda x: True,
         )
         self.check_alert_rules_patcher.start()
-        self.harness = Harness(LokiOperatorCharm)
 
-        # GIVEN this unit is the leader
+        self.harness = Harness(LokiOperatorCharm)
+        self.addCleanup(self.harness.cleanup)
         self.harness.set_leader(True)
+        self.harness.begin_with_initial_hooks()
 
         # AND a "logging" app joins with several units
```

Comment:
The "AND" doesn't make sense here after removing the "GIVEN" above.

Change to..?
```diff
-# AND a "logging" app joins with several units
+# GIVEN a "logging" app joins with several units
```